angular/bootstrap/src/components/modal/modal.component.ts

```diff
@@ -298,11 +298,11 @@ export class ModalComponent<Data> extends BaseWidgetDirective<ModalWidget<Data>>
 
 	ngAfterContentChecked(): void {
 		this._widget.patchSlots({
-			slotDefault: this.slotDefaultFromContent?.templateRef,
+			children: this.slotDefaultFromContent?.templateRef,
 			slotFooter: this.slotFooterFromContent?.templateRef,
 			slotHeader: this.slotHeaderFromContent?.templateRef,
 			slotStructure: this.slotStructureFromContent?.templateRef,
 			slotTitle: this.slotTitleFromContent?.templateRef,
-		});
+		} as any);
```

Comment:
Workaround due to Typescript behavior when filerting keys by their generic value...
After spending too much time trying to find a cleaner way to do this, am adding this little dirty trick.

Any better suggestion is welcome !